src/Dommel/Resolvers.cs

```diff
@@ -78,15 +78,6 @@ public static IEnumerable<ColumnPropertyInfo> Properties(Type type)
 
             return properties;
         }
-        /// <summary>
```

Comment:
This is a breaking change! I don't think we should remove this method. There are probably some applications that use it.